docs/man/docker-events.1.md

```diff
@@ -37,6 +37,10 @@ and Docker images will report:
 **--until**=""
    Stream events until this timestamp
 
+The `--since` and `--until` parameters can be Unix timestamps, RFC3339
```

Comment:
This section needs to conform to what you are doing in the docker-logs.md file and the other files.  

With that change LGTM and we can merge